### PR TITLE
Update BigCommerce

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: entries/b/bigcommerce.com.json [hunk 1]
```diff
@@ -1,17 +1,26 @@
 {
-    "BigCommerce": {
-      "domain": "bigcommerce.com",
-      "url": "https://www.bigcommerce.com",
```
Comment (ApCoder123) [inline entries/b/bigcommerce.com.json:4]: The `url` field is needed in this case, as the `url.rb` tests are failing

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Comment by phallobst** [comment]
It's not really `retail`, is it?

**Comment by 40504** [comment]
> It's not really `retail`, is it?

I don't know which category to include 🤷🏻‍♂️

**Comment by ApCoder123** [comment]
> It's not really `retail`, is it?

@phallobst Do you think that other similar services should be moved out of the `retail` category? E.g. Shift4Shop and Shopify

**Comment by phallobst** [comment]
Maybe a `commerce` category would be useful. This could include some more entries:
2checkout, clover, refersion, revenuecat, spring, thrivecart, tito

**Review by phallobst — APPROVED** [review]

**Pull request merged**